src/licenses/opensource.toml

```diff
@@ -0,0 +1,68 @@
+name = "opensource"
+description = "Une description de l'innitiative Open Source."
+
+[[embeds]]
+title = "Initiative Open Source"
```

Comment:
Open-source

Reply:
It references the full name of the OSI organization, which does not include a dash between "Open" and "Source"